public/index.php

```
@@ -86,6 +94,36 @@ function optionalEnv(string $name): ?string
+    if (! is_int($normalizedPort) || $normalizedPort <= 0) {
+        return new NullAssetStatusCache();
+    }
+
+    $cache = new NullAssetStatusCache();
+
+    try {
+        $cache = RedisAssetStatusCache::fromConnectionConfiguration(
+            $host,
+            $normalizedPort,
+            optionalEnv('REDIS_PASSWORD'),
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Infrastructure\Processing\RedisAssetStatusCache' in method 'assetStatusCache'. (undefined)

(StaticAccess)